### PR TITLE
Port Legal pages to Protocol (Fixes #7441)

### LINKED ISSUE #7441: Port Legal pages to Protocol
## Description

The main `/legal/` pages are still built on Sandstone / Less. They can probably be ported pretty easily using the basic article template style.

https://www.mozilla.org/en-US/about/legal/

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Legal pages are updated in production using Protocol.

#### Tasks
- [ ] Update templates.
- [ ] Remove old assets.

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Port legal pages to use Protocol.
- Sandstone/Less no longer used.
- Unused assets/files removed.

## Issue / Bugzilla link
#7441

## Testing

- [ ] http://localhost:8000/about/legal/
- [ ] http://localhost:8000/about/legal/terms/firefox/
- [ ] http://localhost:8000/about/legal/terms/mozilla/
- [ ] http://localhost:8000/about/legal/terms/services/
- [ ] http://localhost:8000/about/legal/fraud-report/
- [ ] http://localhost:8000/about/legal/firefox/
- [ ] http://localhost:8000/about/legal/terms/vpn/
- [ ] http://localhost:8000/about/legal/terms/firefox-screenshotgo/
- [ ] http://localhost:8000/about/legal/terms/firefox-reality/
- [ ] http://localhost:8000/about/legal/terms/thunderbird/
- [ ] http://localhost:8000/about/legal/terms/firefox/
- [ ] http://localhost:8000/about/legal/acceptable-use/
- [ ] http://localhost:8000/about/legal/report-infringement/
- [ ] http://localhost:8000/about/legal/eula/
- [ ] http://localhost:8000/about/legal/eula/firefox-2/
- [ ] http://localhost:8000/about/legal/eula/firefox-3/
- [ ] http://localhost:8000/about/legal/eula/thunderbird-1.5/
- [ ] http://localhost:8000/about/legal/eula/thunderbird-2/
- [ ] http://localhost:8000/de/about/legal/impressum/

### CODE REVIEW

**Comment by alexgibson** [comment]
The default blockquote styling at the bottom of this page is pretty huge. Perhaps we should consider a smaller variation for these pages?

image

File: bedrock/legal/templates/legal/impressum.html [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
 {% endblock %}
 
 {% block side_extra %}
-  <section>
-    <p>Beratung zu unserer Software finden Sie auf unserer Support-Website unter <a href="https://support.mozilla.org/de/">https://support.mozilla.org/de/</a>.</p>
-    <p>Haben Sie Feedback zu Mozilla Software? Wir freuen uns, von Ihnen zu hören, unter <a href="https://input.mozilla.org/de/feedback">https://input.mozilla.org/de/feedback</a>.</p>
-  </section>
+<section class="mzp-c-sidemenu-main" id="sidebar-menu">
+  <ul>
+    <li><a href="https://support.mozilla.org/de/">https://support.mozilla.org/de/</a></li>
```
Comment (amychurchwell) [inline bedrock/legal/templates/legal/impressum.html]: TODO: reach out to German-speaker for titles for these links. Originals were very long.

File: bedrock/legal/templates/legal/docs-base.html [hunk 2]
```diff
@@ -18,9 +18,19 @@
...
 {% block side_nav %}
-<nav class="mzp-c-sidemenu mzp-c-sidemenu-main">
-  <ul>
-    <li><a href="{{ url('legal.index') }}">{{ _('Back to Legal') }}</a></li>
-  </ul>
+<nav class="mzp-c-sidemenu">
+  <section class="mzp-c-sidemenu-summary mzp-js-toggle" aria-controls="sidebar-menu">
+    <h3 class="mzp-c-sidemenu-label">{{ self.page_title() }}</h3>
+    <ul>
+      <li class="mzp-is-current">{{ self.page_title() }}</li>
```
Comment (alexgibson) [inline bedrock/legal/templates/legal/docs-base.html]: The page title is repeated three times in the side menu on most pages, which feels perhaps a little too much?

image

Perhaps just one title, plus the "Back to Legal" link should be enough? I don't think there's all that much to be gained by the pages linking to themselves?

File: bedrock/legal/templates/legal/impressum.html (hunk 1 above)
Comment (amychurchwell) [reply bedrock/legal/templates/legal/impressum.html]: Received translations for these links from Raphael! 🎉

**Pull request merged**